charts/kubernikus-system/charts/prometheus/templates/_prometheus.yaml.tpl

```diff
@@ -241,7 +241,44 @@ scrape_configs:
       target_label: __metrics_path__
       replacement: /api/v1/nodes/${1}:4194/proxy/metrics
 
-# Static Targets
+- job_name: 'blackbox-ingress'
+  metrics_path: /probe
+  params:
+    # Look for a HTTP 200 response per default.
+    # Can be overwritten by annotating the ingress resource with the expected return codes, e.g. `prometheus.io/probe_code: "4xx"`
+    module: [http_2xx]
```

Comment:
So, this is a default in case the annotation hasn't been overwritten?

Reply:
Yes. Per default a good response is `2xx`. Anything else has to be specified via the annotation on the ingress